libfs/constants.go

```diff
@@ -50,3 +50,12 @@ const FlushJournalFileName = ".kbfs_flush_journal"
 // DisableJournalFileName is the name of the journal-disabling
 // file. It can be reached anywhere within a top-level folder.
 const DisableJournalFileName = ".kbfs_disable_journal"
+
```

Comment:
`libfs` is for code that's shared between dokan and libfuse, so I think these macOS-specific constants belong more in libfuse.